contributing/code/standards.rst

```diff
@@ -167,6 +167,29 @@ Service Naming Conventions
 
 * A group name uses the underscore notation.
 
+.. _contributing-code-standards-comparisons:
+
+Comparisons
+-----------
+
+Use `identical comparison`_ when the expected value must match a specific type:
```

Comment:
I asked the same previously (which disappeared when moving the note to a different page)